src/assets/index.js

```diff
@@ -2,5 +2,12 @@ import brandBanner from "./brand-banners.jpg"
 import menShoe from "./mens-shoe.jpg"
 import womenShoe from './women-shoe.jpg'
 import kidShoe from './kidshoe.jpg'
+import imgShoe1 from './shoe.jpg'
```

Comment:
You can directly do export here, you don't need to use import

Reply:
from this way I comfortable
this reduce line of code and easy to understand that from where i am importing our code